2016/meta/emails/mentor_onboarding.md

```diff
@@ -0,0 +1,7 @@
+Hey %%name%%,
+
+Glad to hear you are interested in mentoring for Hack Camp. Here are the details:
+
+Hack Camp is running from %%camp dates%%, 8:40AM to 6PM. We're hosted at [redacted].
+
+You'll be expected to have the technical knowledge to answer hackers' questions about the workshops. Please familiarize yourself with the content in each workshop before we go over it in camp (https://github.com/hackclub/hackcamp/blob/master/2016/playbook/ACTIVITIES.md#list-of-workshops)
```

Comment:
We need to add Dodge in the list of workshops.

Also could we say something like, "feel free to email me back if you have any questions on the workshops" or something like that. "familiarize" is a pretty vague term